examples/resource-methods/resource_methods.md

```diff
@@ -1,12 +1,14 @@
 # Resource methods
 
-Service objects use `remote` methods to expose services in procedural style: remote methods are named by verbs. <br/><br/>
-Service objects use `resource` methods to expose services in an RESTful style: resources are named by nouns. <br/><br/>
-Resources are motivated by HTTP, but are general enough also to work for GraphQL. 
-`resource` methods are a network-oriented generalization of OO getter/setter concept.
+Service objects use `remote` methods to expose services in procedural style. Remote methods are named by verbs.
+Service objects use `resource` methods to expose services in an RESTful style. Resources are named by nouns. Resources are motivated by HTTP, but are general enough also to work for GraphQL. `resource` methods are a network-oriented generalization of OO getter/setter concept.
```

Comment:
```suggestion
Service objects use `resource` methods to expose services in a RESTful style. Resources are named by nouns. Resources are motivated by HTTP but are general enough to work for GraphQL. `resource` methods are a network-oriented generalization of the OO `getter/setter` concept.
```
Better to say what is `OO` here and have the abbreviation in front within brackets.